src/components/Blasts/BlastVisual.js

```diff
@@ -201,61 +202,92 @@ const styles = theme => ({
   }
 });
 
-function BlastVisual({ classes }) {
-  function CurrentDate() {
-    const date = new Date();
-    const today = date.toLocaleTimeString();
-    return today;
+function CurrentDate() {
+  const date = new Date();
+  const today = date.toLocaleTimeString([], {
+    hour: '2-digit',
+    minute: '2-digit'
+  });
+  return today;
+}
+
+class BlastVisual extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { blastData: [] };
   }
-  return (
-    <div className={classes.root}>
-      <div className={classes.hud}>
-        <img alt="" src={hud} className={classes.hudImage} />
-        <div className={classes.hudCore}>
-          <Typography className={classes.title}>26</Typography>
-          <Typography className={classes.caption}>Recorded Blasts</Typography>
-        </div>
-      </div>
-      <div>
-        <div className={classes.currentTimeLine} />
-        <div className={classes.lastUpdateTimeLine} />
-      </div>
-      <div className={classes.outerDetails}>
-        <div className={classes.socialDetails}>
-          <Typography gutterBottom className={classes.socialText}>
-            Share
-          </Typography>
-          <div className={classes.socialDetailsIcons}>
-            <TwitterShareButton
-              className={classes.shareIconStyle}
-              url="https://alpha.seasensors.africa"
-              title="Mapping blast fishing along the coast of East Africa"
-            >
-              <FontAwesomeIcon icon={faTwitter} size="2x" />
-            </TwitterShareButton>
-            <FacebookShareButton
-              title="Sea Sensors Africa"
-              className={classes.shareIconStyle}
-              url="https://alpha.seasensors.africa"
-              quote="Mapping blast fishing along the coast of East Africa"
-            >
-              <FontAwesomeIcon icon={faFacebookF} size="2x" />
-            </FacebookShareButton>
+
+  componentDidMount() {
+    Tabletop.init({
+      key: '1DTK8hyI1sGooGOU-fQh-amYGc7iJk4Bc6QlARfXuE8U',
+      callback: googleData => {
+        this.setState({ blastData: googleData });
+      },
+      simpleSheet: true
+    });
+  }
+
+  render() {
+    const { classes } = this.props;
+    const { blastData } = this.state;
+
+    const lastArray = blastData[blastData.length - 1];
+    const date = lastArray.Date;
+    const d = date.split(' ')[0];
```

Comment:
```javascript
...
    const lastBlast = blastData[blastData.length - 1];
    const datetime = lastBlast.Date;
    const date = datetime.split(' ')[0];
....
```

May need to check _if_ we actually have `datetime` and default to _some_ value *if* we don't.